src/NanoIoC.Tests/Reset.cs

```diff
@@ -21,7 +21,7 @@ public void ResettingShouldClearAllStores()
 			{
 				container.Reset();
 			}
-			catch (Exception e)

```

Comment:
I like to keep the `e`s as it aids debugging if you ever need to hover and see whats what